src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body class="page">
+    <section class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="card-icon"
+        class="card__image"
+      >
+
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+
+      <article class="card__rating">
+        <div class="stars stars--1">
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+        </div>
+
+        <p>
+            Отзывов: 5
+        </p>
+
+      </article>
+
+      <article class="card__price">
+        <p class="card__price-title">Цена:</p>
+
+        <p class="card__price-number">69 999 грн</p>
+      </article>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >
+      купить
+      </a>
```

Comment:
2 spaces needed before "купить" 

```suggestion
      >
        купить
      </a>
```